.github/workflows/ci.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: ci
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  static-check-containers:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test our container definitions
+        run: tools/static_check_containers
```

Comment:
Please make this a makefile target e.g. `test-containers-hadolint` as discussed on #3761 to make it trivial to reproduce locally